Tools/python/PATTupleSelection_cfi.py

```diff
@@ -55,5 +55,5 @@
 
```

Comment:
I don't remember if you've checked--did the electron or muon (or jet) object selections change at all for UL?

Reply:
Agreed! Maybe for leptons we'll want to do some custom things in the displaced cases, but let's try to use the pre-calculated answers by default (and maybe leave the old ones commented out).